public/app/features/dashboard/panel_model.ts

```diff
@@ -95,7 +95,7 @@ export class PanelModel {
   setViewMode(fullscreen: boolean, isEditing: boolean) {
     this.fullscreen = fullscreen;
     this.isEditing = isEditing;
-    this.events.emit('panel-size-changed');
+    this.events.emit('view-mode-changed');
```

Comment:
There's a few other lines in this files which emits the "panel-size-changed" event and that seems like an obsolete event? 
* https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/panel_model.ts#L114
* https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/panel_model.ts#L119

Reply:
Aha :+1: